tutorials/algorithms/09_IQPE.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>There's a typo here. "measurenment" instead of "measurement".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
Thanks! Fixed in 5728dd9